.pre-commit-config.yaml

```diff
@@ -93,6 +93,7 @@ repos:
         description: Clean build dependencies with gazelle
         language: system
         entry: bazel run //:gazelle
+        stages: [push]
```

Comment:
Running on push seems like an odd choice since a commit has already been made.  Does this block the push if there are differences?

Reply:
Yeah, it's like the other formatters. I mostly just changed it because the lint-runner is much more lightweight and doesn't have bazel[isk] installed. I can change it back and add bazel to that image, though.